### DISCUSSION TITLE
Soundtheory Gullfoss - can't edit values via mouse

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#644] [Help]
Hi, seems like the edit of values via mouse does not work in Gullfoss, tried the AU and VST version, but same problem. I can type in only positive values via keyboard though..
My server is on a mac mini running Catalina (with latest official release of AG installed). My AG plugins runs on a Mac Pro with High Sierra installed.
Any help would be appreciated. Thank you!

Best regards,
Peter

**Comment by apohl79** [comment]
Have you tried beta 4?

**Comment by Pete3030** [comment]
I did and indeed, now it's working perfectly ;) Thanks a lot!

**Comment by Pete3030** [comment]
Hello again, sorry, but with latest Beta I cannot edit the values in Gullfoss in Logic X, it still works in Reaper and Ableton Live though.. I am confused..;)

**Comment by CaBuffalo** [comment]
I think I can do in Logic Pro X.

What's exactly the problem?

On Thu, Oct 28, 2021 at 1:15 PM Pete3030 ***@***.***> wrote:

> Hello again, sorry, but with latest Beta I cannot edit the values in
> Gullfoss in Logic X, it still works in Reaper and Ableton Live though.. I
> am confused..;)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/644#discussioncomment-1555277>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BP6W65BQ3BJPSSI3DUJG4NZANCNFSM5AOEE6XQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Pete3030** [comment]
The problem is that I cannot change the values in der upper row of the Gullfoss plugin via dragging with my mouse as it is working within Reaper and Ableton Live. I am on Logic 10.4.8, OS is High Sierra.

**Comment by CaBuffalo** [comment]
I can change up to a certain value by dragging my mouse up and down, for
example, 30%. I will have to go to server to change the setting to further
value if I need more.

On Fri, Oct 29, 2021 at 8:14 AM Pete3030 ***@***.***> wrote:

> The problem is that I cannot change the values in der upper row of the
> Gullfoss plugin via dragging with my mouse as it is working within Reaper
> and Ableton Live. I am on Logic 10.4.8, OS is High Sierra.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/644#discussioncomment-1559637>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BHQG6UVHHA3TLURN3UJLB6ZANCNFSM5AOEE6XQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Pete3030** [comment]
OK, what are your findings in Live or Reaper? For me there it's working fine. It's just in Logic that I cannot edit the values. Strange..

**Comment by CaBuffalo** [comment]
Sorry, I only use Logic. No experience in Live or Reaper.

On Sat, Oct 30, 2021 at 1:11 AM Pete3030 ***@***.***> wrote:

> OK, what are your findings in Live or Reaper? For me there it's working
> fine. It's just in Logic that I cannot edit the values. Strange..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/644#discussioncomment-1562204>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E7NXZ7HAK2ZKKQGWLUJOZDVANCNFSM5AOEE6XQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.
